lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -49,15 +50,15 @@ public TestAzureBlockBlobFileSystemE2E()
                 .Build();
 
             _fileSystem = TangFactory.GetTang().NewInjector(conf).GetInstance<AzureBlockBlobFileSystem>();
-            _container = CloudStorageAccount.Parse(ConnectionString).CreateCloudBlobClient().GetContainerReference(defaultContainerName);
-            _container.CreateIfNotExistsAsync().Wait();
```

Comment:
I'm assuming you removed the container creation during setup so later tests can auto-create the container. Keep in mind this is only necessary for a couple tests. You could have left the original logic and instead deleted the container on the couple tests where needed. This would have limited the complexity of the other tests.